src/components/TaskDetails/DeleteTask.jsx

```diff
@@ -10,9 +14,9 @@ export default class DeleteTask extends React.Component {
   }
 
   handleDelete = () => {
-    const { taskRef, closeTaskDetails } = this.props;
+    const { taskRef, closeTaskDetails, taskDetailsActions } = this.props;
```

Comment:
You can destructure taskDetailsActions too, like this:

`const { taskRef, closeTaskDetails, taskDetailsActions: { deleteTaskDetails } } = this.props;`

and then use it like : `deleteTaskDetails(taskRef);` instead of `taskDetailsActions.deleteTaskDetails(taskRef);`